specification/compute/resource-manager/readme.go.md

```diff
@@ -27,6 +27,22 @@ directive:
         replace(/, 'DummyOrchestrationServiceName'/,'');
 ```
 
+``` yaml $(go) && $(track2)
+license-header: MICROSOFT_MIT_NO_VERSION
+clear-output-folder: false
+module-name: sdk/arm/compute
+module: github.com/Azure/azure-sdk-for-go/$(module-name)
+output-folder: $(go-sdk-folder)/$(module-name)
+credential-scope: https://management.azure.com//.default
```

Comment:
This can be removed.  The proper scope is included in `armcore` which is included via `azure-arm: true`